### DISCUSSION TITLE
Windows phone 8.1 for Moto e 2014

### DISCUSSION THREAD

**Discussion by vizPilot** [aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick#5] [Q&A] [answered]
Please need help me with moto e 2014 xt1021 for install Windows phone 8.1

My moto e 2014 in edl

**Comment by aravindvnair99** [comment, marked as answer]
> Please need help me with moto e 2014 xt1021 for install Windows phone 8.1

@Vicc2008 Please provide more information on the issue you are facing.

**Comment by vizPilot** [comment]
My moto e 2014 is bricked
